### PR TITLE
[typescript/it] Add typescript-it.html.markdown

### PR DESCRIPTION
**Pull request by chris54721**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: it-it/typescript-it.html.markdown [hunk 1]
```diff
@@ -0,0 +1,226 @@
+---
+language: TypeScript
+contributors:
+    - ["Philippe Vlérick", "https://github.com/pvlerick"]
+translators:
+    - ["Christian Grasso", "https://grasso.io"]
+filename: learntypescript-it.ts
+---
```
Comment (divayprakash) [inline it-it/typescript-it.html.markdown]: You need to add the language spec line here for the site builder to pick it up as a translation of an existing document. In its present state, it just replaces the English TypeScript document.

```
lang: it-it
```

**Comment by divayprakash** [comment]
@chris54721 Please make the changes as requested and I'll merge this in ASAP. Thank you for the contribution!

**Comment by chris54721** [comment]
Done!

**Comment by divayprakash** [comment]
@chris54721 Thank you for the contribution!

**Pull request merged**